core/http/api/v2/cluster/routers.go

```diff
@@ -108,18 +108,6 @@ func (api *API) RegisterRoute(engine *gin.Engine) {
 			Method:      http.MethodGet,
 			Pattern:     fmt.Sprintf("/clusters/:%v/outputs", common.ParamClusterID),
 			HandlerFunc: api.GetOutput,
-		}, {
```

Comment:
If these interfaces are going to be deleted directly, you need to consider the order of deploying horizon & horizon-web.